app/models/report/learner.rb

```diff
@@ -93,27 +111,37 @@ def update_answers
     # AU: We'll use a serialized column to store a hash, for now
     answers_hash = {}
     report_util.saveables.each do |s|
-      hash = {:answer => s.answer, :answered => s.answered?, :submitted => s.submitted?, :question_required => s.embeddable.is_required }
-      if s.respond_to?("has_correct_answer?") && s.has_correct_answer? && s.respond_to?("answered_correctly?")
-        hash[:is_correct] = s.answered_correctly?
```

Comment:
the logic for `hash[:is_correct]`  It was recently modified to better support questions that don't have any correct answers. In this branch, the old logic is now on line 131.